### PR TITLE
Portfolio accessibility

### PR DESCRIPTION
**Pull request by jlvFlores**
In this Pull Request, we:
- Page titles - change the title to Jose Luis's Portfolio
- Image text alternative - update the image alternatives
- Text Heading - no issues found
- Color contrast - added brightness to works image
- Resize - Changed PXs to EMs
- Interaction - added labels to text boxes
- Moving content - no issues found
- Media - no issues found
- Basic structure - no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -138,16 +138,16 @@ <h3>Profesional Art Printing Data</h3>
                 </div>
             </div>
         </section>
-        <section class="about">
+        <section id="about" class="about">
             <div class="gen-desc">
                 <h2 class="heading-text">About me</h2>
-                <p class="paragraph-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="paragraph-text">Hello I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
                 <button class="regular-btn">Get My Resume</button>
```
Comment (iambenkis) [inline index.html:145]: - [x] Your button has not had enough contrast of 4.5:1. Please add enough contrast for this button :+1: 

Screenshot from 2023-01-25 00-12-18

Screenshot from 2023-01-25 00-13-08

File: index.html [hunk 2]
```diff
@@ -185,11 +185,14 @@ <h3>Skills</h3>
...
             <p class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
             <form action="https://formspree.io/f/mjvdkwqz" method="post" class="contact-form">
+                <label for="user-name" hidden >Full name</label>
                 <input type="text" maxlength="30" placeholder="Full name" class="input-field" name="user-name" required>
+                <label for="user-email" hidden>Email</label>
                 <input type="email" placeholder="Email" class="input-field" name="user-email" required>
+                <label for="user-text" hidden>Message</label>
                 <textarea name="user-text" maxlength="500" placeholder="Enter Text here" class="textarea-field" required></textarea>
                 <button type="submit" class="submit-btn">Get in touch</button>
             </form>
```
Comment (iambenkis) [inline index.html:198]: - [x] Your button has not had enough contrast of 4.5:1. Please add enough contrast for this button :+1:

Screenshot from 2023-01-25 00-12-51


Screenshot from 2023-01-25 00-13-08 (1)

**Review by iambenkis — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey Team (@ezraermy , @jlvFlores ) :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Basic Structure is good
:heavy_check_mark: Can be maximized upto 200%


Good job on implementing some of the requirements but there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (brainconnect93) [inline index.html:145]: - [ ] As suggested by the previous reviewer, the `Get My Resume` button does not have enough color contrast in the `hover` effect CSS property. I would kindly suggest you work on it and make sure the color contrast is well aligned with the requirement of 4.5:1

The color contrast result 👇 

col-con

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @Team, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jlvFlores) [reply index.html:145]: Hello @brainconnect93, what tool are you using to measure the color contrast, so that I don't commit the same mistake again.

Comment (brainconnect93) [reply index.html:145]: @jlvFlores, kindly use this [tool](https://www.aditus.io/button-contrast-checker/) to check if you are good to go

**Review by franclobo — APPROVED** [review]
Hi @jlvFlores, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 

## Optional suggestions

- Congratulations for working hard trying to solve the issues with the color contrast 💪🏽 

image



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**